recipes-security/refpolicy/refpolicy-mcs-2.%/policy/booleans.conf

```diff
@@ -0,0 +1,12 @@
+#
+# Enable reading of urandom for all domains.
+# 
+# 
+# 
+# 
+# This should be enabled when all programs
+# are compiled with ProPolice/SSP
+# stack smashing protection.  All domains will
+# be allowed to read from /dev/urandom.
+# 
+global_ssp = false
```

Comment:
I think we want `global_ssp = true`.  That's what `policy.booleans.diff` was setting.

Reply:
I thought I commented on that in the commit message, I'll amend that.

`global_ssp` documentation says:
> This should be enabled when all programs are compiled with ProPolice/SSP stack smashing protection.

It does not appear to be the case in our builds (could not find `-fstack-protector*`), which matches the absence of new AVCs with that Boolean set to `false`.

That being said, giving read permissions to `/dev/urandom` to all domains should not be a huge risk either. So I can switch it back.